apps/backend/.env.example

```
@@ -1,4 +1,7 @@
 OPENAI_API_KEY=
 
-DATABASE_URL=
-USE_OPENAI_MODEL_NAME=
\ No newline at end of file
+USE_OPENAI_MODEL_NAME=
+
+# Better-auth
+BETTER_AUTH_SECRET=
+BETTER_AUTH_URL=http://localhost:8787 # Base URL of your app
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)